guide/14-deep-learning/object-detection.ipynb

```diff
@@ -431,7 +431,9 @@
    "source": [
```

Comment:
<p>capitalize Hungarian</p><blockquote>The <strong>Hungarian</strong> algorithm, also known as</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/549/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>